src/gui/wizard/owncloudwizard.cpp

```diff
@@ -302,6 +372,34 @@ void OwncloudWizard::changeEvent(QEvent *e)
 void OwncloudWizard::customizeStyle()
 {
     // HINT: Customize wizard's own style here, if necessary in the future (Dark-/Light-Mode switching)
+    auto theme = Theme::instance();
+    auto wizardPalette = palette();
+
+    wizardPalette.setColor(QPalette::Window, theme->wizardHeaderBackgroundColor());
+
+    wizardPalette.setColor(QPalette::Base, theme->wizardHeaderBackgroundColor());
+    QColor colorBaseDisabled(wizardPalette.color(QPalette::Base));
```

Comment:
@FlexW 

Let's try to improve readability a bit. Frankly speaking, I am a bit overwhelmed, and most likely you will become like that too when reading it later :)

```
const auto colorWindowText = theme->wizardHeaderBackgroundColor();
const auto colorWindowTextDisabled =QColor(colorWindowText.red(), colorWindowText.green(), colorWindowText.blue(), 128);
wizardPalette.setColor(QPalette::Base, colorWindowText );
wizardPalette.setColor(QPalette::Disabled, QPalette::WindowText, colorWindowTextDisabled);

const auto colorText = theme->wizardHeaderTitleColor();
const auto colorTextDisabled = (colorText.red(), colorText.green(), colorText.blue(), 128);
wizardPalette.setColor(QPalette::Text, colorText );
wizardPalette.setColor(QPalette::Disabled, QPalette::Text, QPalette::WindowText, colorTextDisabled);
```
 
 Or, even better, since you are only using 2 colors from the theme

  ```
 const auto wizardHeaderBackgroundColor= theme->wizardHeaderBackgroundColor();
 const auto wizardHeaderBackgroundColorDisabled= tQColor(wizardHeaderBackgroundColor.red(), wizardHeaderBackgroundColor.green(), wizardHeaderBackgroundColor.blue(), 128);
   
 const auto wizardHeaderTitleColor= theme->wizardHeaderTitleColor();
 const auto wizardHeaderTitleColorDisabled= tQColor(wizardHeaderTitleColor.red(), wizardHeaderTitleColor.green(), wizardHeaderTitleColor.blue(), 128);
```

 
 And then you just use the prepared header background/title colors where you need them.

Reply:
Much better:)